server/rooms.ts

```diff
@@ -1368,14 +1399,12 @@ export class GlobalRoomState {
 				title: room.title,
 				desc: room.settings.desc || '',
 				userCount: room.userCount,
+				section: room.settings.section || 'none',
```

Comment:
`|| undefined` would save bandwidth and shouldn't be difficult for the client to parse.

Reply:
the 'none' is for rooms that havent been placed into a section yet, though thinking back this should probably go into client code when building the room list for side servers on old code, so fair